src/contributors/Ashyum.json

```
@@ -0,0 +1,5 @@
+{
+    "name" : "Ashyum kumar",
+    "desc" : "Computer Science & Engineering student and competitive coder",
+    "github" : "https://github.com/Ashyum"
```

```suggestion
    "name": "Ashyum kumar",
    "desc": "Computer Science & Engineering student and competitive coder",
    "github": "https://github.com/Ashyum"
```
Update it accordingly.